opensource/index.md

```diff
@@ -1,17 +1,95 @@
 ---
 description: Overview of contributing
 keywords: open, source, contributing, overview
-title: Open Source at Docker
+title: Contribute to documentation and open source projects
```

Comment:
I think this is just about contributing to documentation. Maybe instead of or in addition to the section pointing to Moby below, you can have a note at the top of the page that points people there if that's what they were looking for.